src/csharp/Grpc.Tools/ProtoCompile.cs

```diff
@@ -322,6 +322,7 @@ protected override string GenerateResponseFileCommands()
             {
                 cmd.AddArg(proto.ItemSpec);
             }
+            cmd.AddSwitchMaybe("error_format", "msvs");
```

Comment:
@jtattermusch where is the source code for protoc.exe? I see it being downloaded from maven: https://github.com/protocolbuffers/protobuf/blob/master/csharp/build_tools.sh#L43 but I don't know where the source code lives. It looks like the error format is not correct so the Error list in VS cannot link to the correct location in code.

Reply:
Sources for protoc.exe are in https://github.com/protocolbuffers/protobuf, but we are actually building protoc.exe ourselves (from the third_party/protobuf git submodule) while building  the native artifacts before we proceed to building the nugets.

https://github.com/grpc/grpc/blob/7095e051f2183b632aac241c27cf6b53b69d50e9/tools/run_tests/artifacts/artifact_targets.py#L301

Reply:
I'll see if I can find the source and fix the format but looks like it'll probably go into the protobuf repo.